_posts/2019-07-31-quarkus-0-20.0-released.adoc

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: 'Quarkus 0.20.0 released'
+date: 2019-07-31
+tags: quarkus release
+author: gsmet
+---
+
+We just released Quarkus 0.20.0.
+
+[WARNING]
+====
+Quarkus 0.20.0 requires GraalVM 19.1.1 so please upgrade your GraalVM installation.
```

Comment:
I would expect this to only be valable if you are building a native executable.

Reply:
I think this is same wording that was used in previous announcements, correct @gsmet ?